cumulusci/tasks/salesforce/CreateCommunity.py

```diff
@@ -26,33 +28,43 @@ class CreateCommunity(BaseSalesforceApiTask):
             "description": "URL prefix for the community.",
             "required": False,
         },
+        "retries": {
+            "description": "Number of times to retry community creation request"
+        },
         "timeout": {
-            "description": "Time to wait, in seconds, for the community to be created",
-            "default": 120,
+            "description": "Time to wait, in seconds, for the community to be created"
         },
     }
 
     def _init_options(self, kwargs):
         super(CreateCommunity, self)._init_options(kwargs)
-        self.options["timeout"] = int(self.options.get("timeout", 120))
+        self.options["retries"] = int(self.options.get("retries", 6))
+        self.options["timeout"] = int(self.options.get("timeout", 300))
 
     def _run_task(self):
-        self.logger.info('Creating community "{}"'.format(self.options["name"]))
-
-        # Before we can create a Community, we have to click through the "New Community"
-        # button in the All Communities setup page. (This does some unknown behind-the-scenes setup).
-        # Let's simulate that without actually using a browser.
-        self.logger.info("Preparing org for Communities")
-        s = requests.Session()
-        s.get(self.org_config.start_url).raise_for_status()
-        r = s.get(
-            "{}/sites/servlet.SitePrerequisiteServlet".format(
-                self.org_config.instance_url
+        community = self._get_community()
+        if community is not None:
+            raise Exception(
```

Comment:
Should we use something more specific like a `TaskOptionsError` (sort of fits) or a `CumulusCIException` here?